core-services/template-deprecation/_allowlist.yaml

```diff
@@ -757,6 +757,9 @@ templates:
         pull-ci-openshift-installer-release-4.8-e2e-aws-rhel8:
           generated: false
           kind: presubmit
+        pull-ci-openshift-network-tools-master-e2e-aws:
```

Comment:
similarly this one also, I don't see a `pull-ci-openshift-debug-network-master-e2e-aws` job, so I am skeptical if this change is right or not.

Reply:
yeah you should remove this, it was presumably generated from the first version of your PR where you used a template

Reply:
thanks for clearing the confusion, I'll remove this.